lib/CModHandler.h

```diff
@@ -177,13 +177,43 @@ class DLL_LINKAGE CModInfo
 		FAILED,
 		PASSED
 	};
+	
+	struct Version
+	{
+		int major = 0;
+		int minor = 0;
+		int patch = 0;
+		
+		Version() = default;
+		Version(int mj, int mi, int p): major(mj), minor(mi), patch(p) {}
+		
+		static Version GameVersion();
+		static Version fromString(std::string from);
+		std::string toString() const;
+		
+		bool compatible(const Version & other, bool checkMinor = false, bool checkPatch = false) const;
+		bool isNull() const;
+		
+		template <typename Handler> void serialize(Handler &h, const int version)

```

Comment:
why is `version` parameter unused?

```suggestion
		template <typename Handler> void serialize(Handler & h, int version)

```

Reply:
That's declaration of basic serialization function, it's unified across all serializable objects

Reply:
version is needed to work with previous save versions